### ISSUE TITLE
[Related specs] Shape Trees

### ISSUE THREAD

**Issue by pietercolpaert** [constraintAutomaton/shape-index-specification#6]
Shape Trees use ShEx in combination with LDP to describe what items in what containers will adhere to what shapes. It also allows you to write shapes into the containers.

https://shapetrees.org/TR/primer/

At this moment you document that `However, the specification suffers from its complexity and the lack of a concrete definition of a subweb ([the specification is somewhat tied to Solid](https://shapetrees.org/)).`
 * I don’t understand why it suffers from complexity. You can simply use validatedBy, right? The largest complexity is in how to manage these shapes, something you keep out of scope
 * Lack a concrete definition of a subweb: I also don’t understand this, as the container is going to contain items that were validated by the shape, thus defining the “subweb” pretty well?

**Comment by constraintAutomaton** [comment]
The primer is out of date and the actual specification is more complicated, but also handle a more vast use case.

> This Primer is undergoing active development. At this time, and until this advisement is removed, it should not be considered as a reliable source of instruction on the use of Shape Trees. 

I guess we simply do not have the same focus between the two specifications. The shape index is intended to be a lightweight formulation for mainly discovery (by defining a decentralized RDF dataset) and query optimization.

Our formalization is more flexible for the definition of the subweb (we handle disjunct domain). I also think they don't handle cases where not each resource is mapped with a shape. Also, the bound of the domain of publication is less well defined in the shape trees specification, but it only makes sense to define it if you support domains that are not fully mapped with shapes, like us. 

Also, they model their publication space as a tree where the hierarchy has importance, whereas we opted for a flat model, which is an, I think, important distinction.

**Comment by pietercolpaert** [comment]
I would rephrase the sentence on shape trees: make sure nobody will be offended and that your arguments make are non-debatable.

**Comment by constraintAutomaton** [comment]
I tried to change the argument. In the current version.
